packages/account/src/Components/trading-assessment/trading-assessment-new-user.tsx

```diff
@@ -30,17 +48,18 @@ const TradingAssessmentNewUser = ({
                 return { ...accumulator };
             }, {});
         }
-        onSubmit(getCurrentStep() - 1, process_form_values, null, goToNextStep, should_override);
+        onSubmit(getCurrentStep() - 1, process_form_values, goToNextStep, null, should_override);
     };
 
     return (
         <TradingAssessmentForm
-            assessment_questions={assessment_questions}
             form_value={value}
             onSubmit={handleSubmit}
             onCancel={handleCancel}
             setSubSectionIndex={setSubSectionIndex}
             disabled_items={disabled_items}
+            should_move_to_next={false}
+            is_independent_section={false}
```

Comment:
Do we need to give false value here? if we don't pass in `should_move_to_next` and `is_independent_section` by default it will be false